### PR TITLE
wl-gammactl: don't need wlroots

### PR DESCRIPTION
**Pull request by wineee**
## Description of changes

I checked the [upstream code](https://github.com/mischw/wl-gammactl/blob/master/src/main.c) and found that wlroots is not used at all

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff [hunk 1]
```diff
@@ -0,0 +1,20 @@
+diff --git a/meson.build b/meson.build
+index 5253f52..f5ff82e 100644
+--- a/meson.build
++++ b/meson.build
+@@ -6,7 +6,6 @@ project(
```
Comment (teto) [inline pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff:5]: shoudlnt this be applied upstream if they dont use it ?

Comment (wineee) [reply pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff:5]: Yes, I open a pr to upstream https://github.com/mischw/wl-gammactl/pull/15

But the upstream is no longer actively maintained...

Comment (teto) [reply pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff:5]: are you fixing  the package to get rid of an old wlroots derivation or do you actually use the package ? just curious if there is some active fork, anyway last commit is from 2021 so i suppose it's recent enough.

**Review by teto — APPROVED** [review]

File: pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff (hunk 1 above)
Comment (wineee) [reply pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff:5]: I use this package to test my server side implement https://github.com/vioken/waylib/pull/220， maybe I will write a qml application in the future as an alternative

**Pull request merged**